tests/support/Pipelines/Bbh/Test_Inspiral.py

```diff
@@ -98,23 +98,23 @@ def test_inspiral_parameters(self):
         self.assertEqual(params["P"], 5)
         # Control system
         self.assertEqual(params["MaxDampingTimescale"], 20.0)
-        self.assertEqual(params["KinematicTimescale"], 0.4)
-        self.assertAlmostEqual(params["SizeATimescale"], 0.04)
-        self.assertAlmostEqual(params["SizeBTimescale"], 0.04)
-        self.assertAlmostEqual(params["ShapeATimescale"], 2.0)
-        self.assertAlmostEqual(params["ShapeBTimescale"], 2.0)
+        self.assertEqual(params["KinematicTimescale"], 0.2)
+        self.assertAlmostEqual(params["SizeATimescale"], 0.024)
+        self.assertAlmostEqual(params["SizeBTimescale"], 0.016)
```

Comment:
Why are all these different now?

Reply:
they used the horizon masses before, and the target masses now.

Reply:
But these are a factor of 2 different. Surely the horizon masses and target masses can't be that different?

Reply:
These were just the numbers written in the test (horizon masses were 1 and target masses 0.5).

Reply:
Looks like the target masses are 0.6 and 0.4 actually, but yes I see now where the horizon masses were just arbitrarily chosen.